xpk.py

```diff
@@ -263,11 +263,12 @@ class CapacityType(enum.Enum):
         labels:
           xpk.google.com/workload: {args.workload}
       spec:
-        backoffLimit: 0
+        backoffLimit: 4
         completions: {system.vms_per_slice}
         parallelism: {system.vms_per_slice}
         template:
           spec:
+            terminationGracePeriodSeconds: {args.termination_grace_period_seconds}
```

Comment:
I am not sure if this arg is actually added. 

You would need to add this argument, towards the end of this file. For example - https://github.com/google/xpk/blob/main/xpk.py#L6977-L6982

Reply:
The arg is already added here. Thanks for checking!

https://github.com/google/xpk/blob/37fe185d664bc2111526cb1e4276a2a4f27c57f9/xpk.py#L5743-L5751

Reply:
Talked offline, this is part of workload create args!

Reply:
Talked offline, this is part of workload create args!